crt_portal/static/sass/custom/landing.scss

```diff
@@ -465,12 +504,20 @@ $arrow-height: 24px;
   h2 {
     margin-top: 2rem;
     margin-bottom: 1rem;
+    @include at-media-max(mobile-lg) {
+      margin-bottom: 2rem;
+    }
   }
 
   p.crt-landing--text {
     @include text--body-copy__small();
     margin-bottom: 0;
   }
+  @include at-media-max(mobile-lg) {
+    .crt-button--large {
+      margin-top: 0;
+    }
+  }
```

Comment:
👇 Right below this chunk of code is a chunk of code that looks like this:
```
.crt-landing--already_submitted {
  h2 {
    margin-top: 1rem;
  }
}
```
It wasn't changed in the scope of this pull request, but it should be removed.